docker-compose.yml

```diff
@@ -4,12 +4,8 @@
 
 version: '2'
 services:
-  bmo:
+  bmo.dev:
     image: mozillaconduit/docker-bmo
-    ports:
-      - "7777:80"
-    volumes:
-      - bmo-mysql-db:/var/lib/mysql
```

Comment:
I was under the impression the named volume is necessary to make BMO save test data across service restarts.  Is it no longer needed?